README.md

```diff
@@ -104,9 +104,22 @@ Run everything: `bundle exec rake`
 
 When new pages are added to the application, ensure they are added to `./.pa11yci` so that they can be scanned.
 
-### Automatic linting and terraform formatting
-
-To enable automatic ruby linting and terraform formatting on every `git commit` follow the instructions at the top of `.githooks/pre-commit`
```

Comment:
[suggestion] since we're not using this file anymore, let's remove `.githooks/pre-commit`